index.js

```diff
@@ -0,0 +1,73 @@
+var dayTitle = document.querySelector(".t-day"),
+	dateTitle = document.querySelector(".t-date"),
+	monthTitle = document.querySelector(".month-year")
+var today = new Date();
+var days = [
+	'SUN',
+	'MON',
+	'TUE',
+	'WED',
+	'THU',
+	'FRI',
+	'SAT'
+]
+var months = [
+	'JAN',
+	'FEB',
+	'MAR',
+	'APR',
+	'MAY',
+	'JUN',
+	'JUL',
+	'AUG',
+	'SEP',
+	'OCT',
+	'NOV',
+	'DEC'
+]
+var currentDay = today.getDay();
+dayTitle.innerText = `${days[currentDay]}`;
+var currentDate = today.getDate();
+dateTitle.innerText = `${currentDate}`;
+var currentMonth = today.getMonth();
+var currentYear = today.getFullYear();
+monthTitle.innerText = `${months[currentMonth]} ${currentYear}`;
+
+var tableDates = document.querySelectorAll("td");
```

Comment:
days와 months배열 안에서는 홑따옴표를 사용했는데
이후부터는 쌍따옴표를 사용하셨습니다.
코드 스타일 통일해주시면 좋을것같아요
```suggestion
var tableDates = document.querySelectorAll('td');
```